tests/cupyx_tests/scipy_tests/sparse_tests/test_csc.py

```diff
@@ -890,11 +890,20 @@ def test_has_canonical_format4(self, xp, sp):
         assert 2 == len(M.indices)  # unaffected content
         return M
 
+    @testing.with_requires('scipy>1.6.0')
```

Comment:
Maybe `scipy>=1.7`?

Reply:
Won't this SciPy's bug be fixed in 1.6.1?

Reply:
You are right.  The SciPy PR has the `backport-candidate` label.